__tests__/Unit/Components/Tasks/TaskLevelMap.test.tsx

```diff
@@ -62,9 +76,8 @@ describe('TaskLevelMap', () => {
         expect(removeButtons).toHaveLength(2);
 
         fireEvent.click(removeButtons[0]);
-        expect(updateTaskTagLevel).toHaveBeenCalledWith(
-            taskTagLevel[0],
-            'delete'
-        );
+
+        expect(tagElements).toHaveLength(2);
```

Comment:
This test is failing @ankushdharkar, 
PS: added a test for delete API, it is passing

Reply:
fixed now